units/src/locktime/relative.rs

```diff
@@ -157,6 +157,13 @@ impl std::error::Error for TimeOverflowError {}
 mod tests {
     use super::*;
 
+    #[test]
+    fn from_seconds_ceil_with_zero_seconds() {
```

Comment:
I don't think this actually tests any new code paths..

Reply:
i had just added as a sanity test, but you're right. i'll remove this case